documentation/tls-best-practices.md

```diff
@@ -0,0 +1,71 @@
+---
+title: IRCv3 TLS Best Practices
+layout: spec
+copyrights:
+  -
+    name: "William Pitcock"
+    period: "2016"
+    email: "nenolod@dereferenced.org"
+---
+
+This document specifies a series of baseline security practices for managing TLS on IRC networks.
+It should be considered non-normative.
+
+
+# Clients
+
+* Clients SHOULD upgrade the default port to 6697 and enable TLS per IETF [RFC 7194][rfc7194].
+
```

Comment:
This will result in a failure to connect for the second largest IRC network, and thus represent a support overhead for client authors. As such I wouldn't recommend it. We could say something like:

"Clients should prefer TLS connections on default port 6697 if available when populating their preloaded server lists."